### ISSUE TITLE
Sidekiq Web Pagination Broken

### ISSUE THREAD

**Issue by chadrschroeder** [mhenrixon/sidekiq-unique-jobs#309]
When there are more than 100 digests to display on the Sidekiq Web Unique Digests page, the following error occurs:

```
NoMethodError
undefined method `>' for nil:NilClass
```

I think this is because the `@current_page` value is `nil`.  Sidekiq Web [expects this variable in its paging partial](https://github.com/mperham/sidekiq/blob/master/web/views/_paging.erb#L6):

```erb
<%# sidekiq/web/views/_paging.erb %>
<% if @current_page > 1 %>
  ...
```

Also, when are these unique digests purged?  They remain in Redis after my jobs have completed successfully and the `unique:keys` set continues to increase in size.

**Comment by mhenrixon** [comment]
Regarding the first question, should not be a problem with sidekiq/web as I am not paging it yet. Would be cool to know the exact line number it goes wrong. 

Regarding the second question. They should be removed upon completion or when deleted from the sidekiq web extension. BUT... I found a bug, I try to delete the wrong ones. I'll have a fix for that shortly.

**Comment by mhenrixon** [comment]
This line https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/redis/signal.lua#L12 should say: `redis.call('SREM', unique_keys, unique_digest)`

**Comment by chadrschroeder** [comment]
For the first question, the issue is because `@count` defaults to `100` [here](https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/lib/sidekiq_unique_jobs/web.rb#L26) and that causes us to render the paging partial [here](https://github.com/mhenrixon/sidekiq-unique-jobs/blob/master/lib/sidekiq_unique_jobs/web/views/unique_digests.erb#L10).

**Comment by mhenrixon** [comment]
Cool, I'll add support for pagination as well while I'm at it then. Thanks for reporting

**Comment by chadrschroeder** [comment]
Thanks for the quick response.

**Comment by mhenrixon** [comment]
Should be fixed in `v6.0.2` @chadrschroeder! Thanks again for taking the time to report the issue.
